repository-setup.md

```diff
@@ -0,0 +1,9 @@
+# Repository Layout
+
+- Master is always Protected.
+- Review where possible must have a second pair of eyes.
+- Squash merging is up to the repository primary maintainer
+
+We aim to have clean understandable updates, and good git history. Rather than hiding a mess under the carpet.
```

Comment:
Make this a 2nd sentence on the last item in the list. It's a supporting note for that specific item and not all the items